system/jlib/jlog.hpp

```diff
@@ -1483,6 +1483,34 @@ enum LogAccessReturnColsMode
     RETURNCOLS_MODE_all
 };
 
+enum LogAccessMappedField
+{
+    LOGACCESS_MAPPEDFIELD_timestamp,
+    LOGACCESS_MAPPEDFIELD_jobid,
+    LOGACCESS_MAPPEDFIELD_component,
+    LOGACCESS_MAPPEDFIELD_class,
+    LOGACCESS_MAPPEDFIELD_audience,
+    LOGACCESS_MAPPEDFIELD_instance,
+    LOGACCESS_MAPPEDFIELD_host,
+    LOGACCESS_MAPPEDFIELD_unmapped
+};
+
+enum SortByDirection
+{
+    SORTBY_DIRECTION_none,
```

Comment:
This doesn't seem to have much meaning.  Why not default to ascending and also avoid extra logic elsewhere?

Reply:
It didn't seem the interface should force a specific behavior. It feels the implementations should control default behavior.

Reply:
I don't really understand the argument.  It isn't optional in the request.

As for cascading issues, if it is not specified and the format isn't specified then then I think you will generate invalid json.